extension/src/panel.ts

```diff
@@ -88,39 +88,52 @@ export class HistoryDebuggerPanel {
   private constructor(
     protected readonly extensionUri: vscode.Uri,
     private readonly secretStorage: vscode.SecretStorage,
-    protected readonly httpServerUrl: string,
+    protected readonly server: Server,
   ) {
-    // TODO: if "getting started" page is on the right, replace it
-    this.panel = vscode.window.createWebviewPanel(HistoryDebuggerPanel.viewType, "Temporal", vscode.ViewColumn.Beside, {
-      // Enable javascript in the webview
-      enableScripts: true,
-
-      // And restrict the webview to only loading content from our extension's compiled directory.
-      localResourceRoots: [vscode.Uri.joinPath(extensionUri, "webview/dist")],
-    })
+    const column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined
```

Comment:
Based on our zoom review I'm not sure if this is better than always opening `Beside`. Maybe leave it as it was for now?

Reply:
It will still always open the panel `Beside`, but now it checks for an active panel and places it beside (or just to the right of) that panel. I think this is slightly better, but maybe only slightly 😅

Reply:
I think `Beside` might be better based on our zoom review where we saw the panel disappear but I don't remember when that happens and if it's related to this change.

Reply:
That issue seems to be unrelated to this change. I have been unable to reproduce it with the latest changes (or reverting this change) and I think it was likely due to where we were using `.show()` for the panel.